transcom/mymove
# updated aoa sit days

## [B19786](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A946165)

## Summary

This PR updates the days in SIT to be inclusive in the AOA Packet.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Make any move that has an AOA
2. Login as a SC
3. Edit and submit move
4. Refresh page
5. Click "Download AOA Packet"
6. Open downloaded PDF and confirm total days in sit are inclusive
*Example if sit days are April 15 - April 19; confirm Days: 5

### Backend


## Screenshots

<img width="788" alt="Screenshot 2024-05-07 at 6 08 33 AM" src="https://github.com/transcom/mymove/assets/146969726/597645e5-9859-4df4-ba18-b3fe1f694068">
